tests/system/HTTP/RedirectExceptionTest.php

```diff
@@ -67,8 +68,10 @@ public function testResponseWithoutStatusCode(): void
 
     public function testLoggingLocationHeader(): void
     {
+        Time::setTestNow('2023-11-25 12:00:00');
```

Comment:
Whenever you set the time, don't forget to reset the current time.
I recommend you add tearDown() like LoggerTest.php.

Reply:
Done.